### PR TITLE
added broadway-bundle & event-store-dbal recipes

### PR DESCRIPTION
**Pull request by DerStoffel**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by DerStoffel** [comment]
waiting for https://github.com/broadway/broadway-bundle/pull/39

**Comment by DerStoffel** [comment]
Pull Request on broadway/broadway-bundle has been approved and merged.
Is there any way to trigger symfony-bot again?

**Comment by fabpot** [comment]
Let me see what I do to automate that :)

**Comment by fabpot** [comment]
We need a new release to make validation pass?

**Comment by DerStoffel** [comment]
Finally, things went through 🥇

File: broadway/broadway-bundle/0.3/config/packages/broadway.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+broadway: ~
```
Comment (fabpot) [inline broadway/broadway-bundle/0.3/config/packages/broadway.yaml]: We don't add "empty" config files to avoid cluttering the config directory.

Comment (DerStoffel) [reply broadway/broadway-bundle/0.3/config/packages/broadway.yaml]: are those rules somewhere written down? Did I miss this?

Comment (fabpot) [reply broadway/broadway-bundle/0.3/config/packages/broadway.yaml]: I'm going to add an auto-validation for this if possible and a note in the README.

**Pull request merged**